platformio_esp32s3_envs.ini

```diff
@@ -119,10 +119,74 @@ build_flags               = ${esp32s3_common.build_flags}
+                            -DFEATURE_ARDUINO_OTA=1
+                            -DPLUGIN_BUILD_CUSTOM
+                            -DFEATURE_SD=1
+extra_scripts             = ${esp32s3_common.extra_scripts}
+                            pre:tools/pio/pre_custom_esp32.py
+board_build.partitions    = esp32_partition_app3520k_spiffs1088k.csv
+
+[env:custom_ESP32s3_8M1M_LittleFS_OPI_PSRAM_CDC]
+extends                   = env:custom_ESP32s3_8M1M_LittleFS_CDC
+board                     = esp32s3cdc-qio_opi
+board_name                = "Espressif Generic ESP32-S3 8M Flash OPI PSRAM, ESPEasy 3520k Code/OTA 1088k FS"
+
+
+[env:max_ESP32s3_8M1M_LittleFS_CDC]
+extends                   = esp32s3_common
+board                     = esp32s3cdc-qio_qspi
+board_upload.flash_size   = 8MB
+board_name                = "Espressif Generic ESP32-S3 8M Flash, ESPEasy 3520k Code/OTA 1088k FS"
+lib_ignore                = ${esp32_always.lib_ignore}
+                            ESP32_ping
+build_flags               = ${esp32s3_common_LittleFS.build_flags}  
+                            -DFEATURE_ARDUINO_OTA=1
+                            -DPLUGIN_BUILD_MAX_ESP32
+                            -DPLUGIN_BUILD_IR_EXTENDED
+extra_scripts             = ${esp32_common.extra_scripts}
+board_build.partitions    = esp32_partition_app3520k_spiffs1088k.csv
+
+
+[env:max_ESP32s3_8M1M_LittleFS_OPI_PSRAM_CDC]
+extends                   = env:max_ESP32s3_8M1M_LittleFS_CDC
+board                     = esp32s3cdc-qio_opi
+board_name                = "Espressif Generic ESP32-S3 8M Flash OPI PSRAM, ESPEasy 3520k Code/OTA 1088k FS"
+
+
+[env:custom_ESP32s3_16M8M_LittleFS_CDC]
+extends                   = esp32s3_common
+board                     = esp32s3cdc-qio_qspi
+board_upload.flash_size   = 16MB
+board_name                = "Espressif Generic ESP32-S3 16M Flash, ESPEasy 4096k Code/OTA 8124k FS"
+lib_ignore                = ${esp32_always.lib_ignore}
+                            ESP32_ping
+build_flags               = ${esp32s3_common_LittleFS.build_flags} 
+                            -DFEATURE_ARDUINO_OTA=1
+                            -DPLUGIN_BUILD_CUSTOM
+                            -DPLUGIN_BUILD_IR_EXTENDED
+                            -DFEATURE_SD=1
+extra_scripts             = ${esp32s3_common.extra_scripts}
+                            pre:tools/pio/pre_custom_esp32.py
+board_build.partitions    = esp32_partition_app4096k_spiffs8124k.csv
+
+[env:custom_ESP32s3_16M8M_LittleFS_OPI_PSRAM_CDC]
+extends                   = env:custom_ESP32s3_16M8M_LittleFS_CDC
+board                     = esp32s3cdc-qio_opi
+board_name                = "Espressif Generic ESP32-S3 16M Flash OPI PSRAM, ESPEasy 4096k Code/OTA 8124k FS"
+
+
 [env:max_ESP32s3_16M8M_LittleFS_CDC]
 extends                   = esp32s3_common
 board                     = esp32s3cdc-qio_qspi
 board_upload.flash_size   = 16MB
+board_name                = "Espressif Generic ESP32-S3 16M Flash, ESPEasy 4096k Code/OTA 8124k FS"
```

Comment:
board name should be part of the board definition.

Reply:
Fixed.